manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-link",
-  "version": "0.6.2",
+  "version": "0.8.0-beta.0",
```

Comment:
Don't forget to change this back before merging ;)

Reply:
Ok!